### DISCUSSION TITLE
Wrong output for JS code

### DISCUSSION THREAD

**Discussion by 0xGSAT** [community/community#54607] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

So, I was using Copilot to help me auto-complete some of the notes in JS. That's when I came across a suggestion by copilot which was wrong! Below is the screen shot.

Explaination:
1.1 + 1.3 in JS will equal to ~2.4000004  so the condition will be False and hence there should be no output but copilot suggest the expected output to be True

**Note:** Apologies in advance if I didn't follow any convention or something. This is my first time opening a discussion. But I did tried to search for a similar issue before posting this.

<img width="526" alt="copilot bug" src="https://user-images.githubusercontent.com/104596776/236608673-ab10ae06-638f-457f-9a2a-bd4f3436bcbf.png">

**Comment by emilburganov** [comment]
This is a feature of JS 0.1 + 0.2
https://blog.luismarques.io/why-01-02-03-in-javascript-and-how-to-fix-it#heading-using-decimaljshttpswwwnpmjscompackagedecimaljs
